src/client/graphql/QuoteCart.graphql

```diff
@@ -2,6 +2,9 @@ query QuoteCart($id: ID!, $locale: Locale!) {
   quoteCart(id: $id) {
     id
     bundle {
+      quotes {
```

Comment:
I'm now requesting `QuoteCart.bundle.quotes` because of the pre-requisites of the project is displaying the options in the following order: _Home_, _House_, _Travel_ and _Accident_ and I've saw that this field is populated by giraffe with this sorting logic.

Before we were [getting all the quotes based on the possible variations field](https://github.com/HedvigInsurance/web-onboarding/pull/1207/files#diff-796ae58a1a2f879ba80b3f21d7c428929fcdde7975a2a208baf2871c8c2fb15aL70). That was resulting in a list of quotes on the wrong order. I'm avoiding to sort twice over increasing the QuoteCart payload 🤷🏻

Reply:
Sure, I guess we can assume for these countries that we sell everything standalone from now